gdi/opentelemetry/automatic-discovery/linux/linux-backend.rst

```diff
@@ -287,15 +287,15 @@ Using the installer script, you can install and activate automatic discovery for
 
     .. tab:: .NET
 
-        Using the installer script, you can install and activate automatic discovery for .NET for either all supported .NET applications on the host via the system-wide method or for only .NET applications running as ``systemd`` services.
-
-        .. note:: By default, automatic discovery is activated for all languages (Java, Node.js, and .NET) when using the installer script. To deactivate automatic discovery for other languages, add the ``--without-instrumentation-sdk [language]`` option in the installer script command.
-
         .. tabs::
 
             .. tab:: Installer script 
 
-                .. tabs:: 
+                Using the installer script, you can install and activate automatic discovery for .NET for either all supported .NET applications on the host via the system-wide method or for only .NET applications running as ``systemd`` services.
+
+                .. note:: By default, automatic discovery is activated for all languages (Java, Node.js, and .NET) when using the installer script. To deactivate automatic discovery for other languages, add the ``--without-instrumentation-sdk [language]`` option in the installer script command.
```

Comment:
@jeffreyc-splunk - should this be 

By default, automatic discovery is activated for all languages (Java, Node.js, and .NET) when using the installer script. To deactivate automatic discovery for one of the languages, add the ``--without-instrumentation-sdk [language]`` option in the installer script command.

Reply:
This was just moved to under the `.NET/nstaller script` tab without any content changes, and looks like this note is the same for Java and Node.js.

Technically, the `--without-instrumentation-sdk` option supports one or more languages, e.g. `--without-instrumentation-sdk java,nodejs` to opt-out of Java and Node.js.  Should all of these notes in this file be updated to clarify this option and supported values?

Reply:
Alternatively, the `--with-instrumentation-sdk [language]` option would activate only the specified language and deactivate all other languages.

Reply:
What I meant is the original note makes it seem like there's more languages than Java/Node/Net. It's just a wording issue. I'll merge and think about it. Thanks so much for your help! :)